src/sincosint.jl

```diff
@@ -18,7 +18,7 @@ using Base.Math.@horner
 function sinint(x::Float64)
     t = x*x
     if t ≤ 36.0
-        return x * @horner(t, 1.00000000000000000000E0,
```

Comment:
ok thanks for letting me know, I opened a PR to go back. Coverage information is typically a little noisy and not always exact.